backend/apps/accounts/views.py

```
@@ -583,642 +103,163 @@ def post(self, request):
             )
 
         user = reset_token.user
+        
+        # Set new password (this will trigger JWT invalidation via signal)
         user.set_password(new_password)
+        
+        # Update last_password_change in profile
         if hasattr(user, "profile"):
             user.profile.last_password_change = timezone.now()
```

Ruff (0.15.20)

[error] Undefined name `timezone`

(F821)